src/components/Training/module-2/components/module2-3.js

```diff
@@ -147,10 +188,14 @@ const MealTipsWrapper = () => {
               <li>
                 <p>
                   If you and the foster parent decide to make arrangements for
-                  you providing babysitting or transportation, the foster
-                  parent’s agency or county may require a background check,
-                  which can take up to three weeks. We are here to help with
-                  that process. Email connect@fostertogether.co to get started.
+                  you providing babysitting or transportation, the{" "}
```

Comment:
"Tips for Those Who Want to Go Beyond the Meal" is larger and bolded in the figma document